README.md

```diff
@@ -1,6 +1,9 @@
 # Devito: Fast Finite Difference Computation from Symbolic Specification
 
-![Build Status](https://travis-ci.org/devitocodes/devito.svg?branch=master)
+![Build Status for the Core backend](https://github.com/devitocodes/devito/workflows/CI-core/badge.svg)
+![Build Status for the OPS backend](https://github.com/devitocodes/devito/workflows/CI-ops/badge.svg)
+![Build Status for the YASK backend](https://github.com/devitocodes/devito/workflows/CI-yask/badge.svg)
+![Build Status with MPI](https://github.com/devitocodes/devito/workflows/CI-mpi/badge.svg)
```

Comment:
Note: maybe a bit much but this way we have badges for each backend + mpi, doesn't take much space

Reply:
I suggest however to remove the OPS and YASK ones

I like CI-core and CI-mpi

Reply:
force-push to master once landed is fine

Reply:
Can do that, also will change badges so that clicking on it actually send you to the CI not just the image of the badge